src/Components/SearchBar/SearchBar.jsx

```diff
@@ -8,7 +8,7 @@ const SearchBar = ({ value, onChange }) => {
     <input
       className="search-bar"
       type="text"
-      placeholder="Search Organisations"
+      placeholder="Search"
```

Comment:
I would suggest passing the Placeholder as a prop, so that we can have dynamic placeholder for different search bar's

Reply:
done that